src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -743,15 +741,13 @@ public static unsafe IntPtr StringToCoTaskMemUni(string? s)
             // Overflow checking
             if (nb < s.Length)
             {
-                throw new ArgumentOutOfRangeException(nameof(s));
+                ThrowHelper.ThrowArgumentOutOfRangeException(ExceptionArgument.s);
             }
 
             IntPtr hglobal = AllocCoTaskMem(nb);
```

Comment:
Nit: `hglobal` name is copy&paste from the StringToHGlobalUni method. It does not make sense here. It would be nice to rename this to e.g. `ptr`.